src/cdsetool/query.py

```diff
@@ -42,9 +42,10 @@ class FeatureQuery:
 
     total_results = None
 
-    def __init__(self, collection, search_terms):
+    def __init__(self, collection, search_terms,proxies = {}):
```

Comment:
Try to run `black src` from the top directory again, I'm pretty sure there's a missing space between `,` and `proxies`.

Reply:
added all the spaces after the , in the proxy that I added